src/notation/internal/notationinteraction.cpp

```diff
@@ -5460,6 +5466,7 @@ void NotationInteraction::addLyricsVerse()
     score()->endCmd();
 
     score()->select(lyrics, SelectType::SINGLE, 0);
+    lyrics->setFontStyle(fStyle);
```

Comment:
Why not a few lines furter up, before `score()->undoAddElement(lyrics);`?

Reply:
And make it `lyrics->setFontStyle(oldLyrics->fontStyle())`?